src/common/GurobiWrapper.cpp

```diff
@@ -93,7 +93,6 @@ void GurobiWrapper::resetModel()
 
 void GurobiWrapper::reset()
 {
-    _nameToVariable.clear();
     _model->reset();
```

Comment:
@idan0610 , the fix is to simply not clear this map when calling reset(), which is intended to reset the solver without removing the constraints.